src/applications/personalization/profile-2/tests/e2e/profile.mpi-error.cypress.spec.js

```diff
@@ -29,7 +29,13 @@ function test(mobile = false) {
 
   // Should show an error alert about not being able to connect to MPI
   // TODO: We might show a different error message in this particular case since in this case we can't connect to MPI.
-  cy.findByText(/We can’t match your information to our Veteran records/i)
+  cy.findByText(/We can’t access your Veteran records/i)
```

Comment:
the change in the component rendering logic means that this test now shows the "can't connect to MPI" error.